src/collections/blog/2023/2023-06-30-Introduction-layer5-learning-paths/index.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: " Introducing Layer5 Learning Paths"
+subtitle: " Introducing Layer5 Learning Paths"
+date: 2020-06-23 10:30:05 -0530
+author: Akshaya
+thumbnail: ./gsoc-wide.webp
+darkthumbnail: ./gsoc-wide.webp
+category: "community"
+description: "This is a short summary of my blog post. This text will show in the blog's list view and in community newsletters."
+tags:
+ - Community
+ - Projects
+ - News
+type: Blog
+resource: false
```

Comment:
instead of resource it should be featured and value of featured and published must be true.
featured: false
published: true